samples/extensions.csproj

```diff
@@ -5,7 +5,7 @@
 	<DefaultItemExcludes>**</DefaultItemExcludes>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="1.8.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.0.0-alpha" />
```

Comment:
I think upgrading this to `2.0.0-beta3` would be sufficient. I'm not sure I want to delineate `1.0` vs `2.0` samples, because unlike for C#, all of the `1.0` samples should work in `2.0`. That being said, I wouldn't be opposed with keeping the current samples at `1.8.3`/`1.8.4`, with new entity samples pointing to `2.0.0-beta3`, at least until we GA.